libkbfs/identify_util.go

```diff
@@ -168,7 +168,7 @@ func identifyUser(ctx context.Context, nug normalizedUsernameGetter,
 	} else {
 		reason = fmt.Sprintf("You accessed a private folder with %s.", username.String())
 	}
-	userInfo, err := identifier.Identify(ctx, username.String(), reason)
+	resName, resID, err := identifier.Identify(ctx, username.String(), reason)
```

Comment:
I thought "res" was pretty standard for RPC results, but I'll expand it to "result*".